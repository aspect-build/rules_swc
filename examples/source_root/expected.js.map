{"version":3,"sources":["in.ts"],"sourceRoot":"https://my-website.com/debug/source/","sourcesContent":["export const a: string = \"foo\";\n"],"names":["a"],"mappings":"AAAA,OAAO,IAAMA,IAAY,MAAM"}