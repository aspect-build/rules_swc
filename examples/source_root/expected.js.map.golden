{
    "sources": [
        "in.ts"
    ],
    "sourceRoot": "https://my-website.com/debug/source/"
}